lib/stack_master/stack_definition.rb

```diff
@@ -9,7 +9,6 @@ class StackDefinition
                   :notification_arns,
                   :base_dir,
                   :template_dir,
-                  :secret_file,
```

Comment:
The stack definition still needs to contain this because the GPG resolver depends on it https://github.com/envato/stack_master-gpg_parameter_resolver/blob/master/lib/stack_master/parameter_resolvers/secret.rb#L43

Reply:
The GPG Parameter Resolver actually adds this back in.

https://github.com/envato/stack_master-gpg_parameter_resolver/blob/5eacf9639941aa830ef0f640ef9a5afb7e76eaf5/lib/stack_master/parameter_resolvers/secret.rb#L4-L6

This is a rather dubious approach though. Thoughts?

Reply:
Actually, I'm preferring to move all the support into the gem. That way it doesn't need to be supported in StackMaster, we can make a clean break in version 2.

Later we could re-work the GPG Parameter Resolver to not use this configuration.

Reply:
Wholly monkey patching that's ugly.

Ok let's leave it as is and refactor for a v1 of GPG parameter resolver.